compiler/src/circuit_design/circuit.rs

```diff
@@ -158,6 +144,9 @@ impl WriteLLVMIR for Circuit {
 
         // Code for the templates
         for t in &self.templates {
+            if cfg!(debug_assertions) {
```

Comment:
This println already happens within the `produce_llvm_ir` call immediately following so it would be duplicated. 
https://github.com/Veridise/circom/blob/19e4b906a86c14be9309249a884ba5523ab8f939/compiler/src/circuit_design/template.rs#L60-L62
You should instead wrap that location with this conditional.

Reply:
Gotcha---must have changed in the rebase.